doc/man3/SHA256_Init.pod

```diff
@@ -49,7 +49,12 @@ L<openssl_user_macros(7)>:
 
 All of the functions described on this page are deprecated.
 Applications should instead use L<EVP_DigestInit_ex(3)>, L<EVP_DigestUpdate(3)>
-and L<EVP_DigestFinal_ex(3)>.
+and L<EVP_DigestFinal_ex(3)>, or L<EVP_Q_digest(3)>.
+SHA1(), SHA224(), SHA256(), SHA384(), and SHA256()
+can continue to be used by defining B<OPENSSL_COMPENSATE_DEPRECATION>.
```

Comment:
OPENSSL_COMPENSATE_DEPRECATION?
Why not just have them always there?

Reply:
Opos, this was I leftover I forgot to remove - done now.
Last week we had decided to go without `OPENSSL_COMPENSATE_DEPRECATION`.